readme.txt

```diff
@@ -152,5 +152,10 @@ A complete documentation is available in the [GitHub repository Wiki](https://gi
 
 For the complete changelog, check out our [GitHub repository](https://github.com/pluginkollektiv/antispam-bee).
 
+== Upgrade Notice ==
+
+= 2.7.1 =
+Please flush your cache, after you have updated.
```

Comment:
I find this sentence to bee a bit clumsy, mainly because of the comma.

How about:
```
This update changes the actual markup of your comment fields. Please make sure that you flush your caches after the plugin was updated!
```